core/lib/zksync_core/src/api_server/web3/namespaces/eth.rs

```diff
@@ -700,21 +571,15 @@ impl EthNamespace {
 
     #[tracing::instrument(skip(self, tx_bytes))]
     pub async fn send_raw_transaction_impl(&self, tx_bytes: Bytes) -> Result<H256, Web3Error> {
-        const METHOD_NAME: &str = "send_raw_transaction";
-
-        let method_latency = API_METRICS.start_call(METHOD_NAME);
         let (mut tx, hash) = self.state.parse_transaction_bytes(&tx_bytes.0)?;
         tx.set_input(tx_bytes.0, hash);
 
         let submit_result = self.state.tx_sender.submit_tx(tx).await;
-        let submit_result = submit_result.map(|_| hash).map_err(|err| {
-            tracing::debug!("Send raw transaction error: {err}");
+        submit_result.map(|_| hash).map_err(|err| {
+            tracing::debug!("Send raw transaction error: {err}"); // FIXME: do we want to track `SubmitTxError`s in other methods?
             API_METRICS.submit_tx_error[&err.prom_error_code()].inc();
```

Comment:
Subj.

Reply:
No opinion here.

Reply:
Removed the FIXME; if we want, it could be dealt with separately.